packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -12683,29 +12683,6 @@ export default{
 				Current Version ID: Galaxy-Class"
 			`);
 		});
-
-		it("should disable observability if not explicitly defined", async () => {
-			writeWranglerConfig({});
-			await fs.promises.writeFile("index.js", `export default {};`);
-			mockSubDomainRequest();
-			mockUploadWorkerRequest({
-				expectedSettingsPatch: {
-					observability: {
-						enabled: false,
-					},
-				},
-			});
-
-			await runWrangler("deploy index.js");
-			expect(std.out).toMatchInlineSnapshot(`
-				"Total Upload: xx KiB / gzip: xx KiB
-				Worker Startup Time: 100 ms
-				Uploaded test-name (TIMINGS)
-				Deployed test-name triggers (TIMINGS)
-				  https://test-name.test-sub-domain.workers.dev
-				Current Version ID: Galaxy-Class"
-			`);
-		});
```

Comment:
Why is this test removed?

Reply:
Just for my understanding: I don't see your changes enabling `observability` by default. So if the users don't have it configured, it should still be `enabled: false`, right?